<launch>
    <arg name="node_name" />
    <arg name="target_model_name"  />
    <arg name="goal_pose_x"   />
    <arg name="goal_pose_y"  />
    <arg name="goal_pose_z"   />
    <arg name="/path_topic"   />
    <arg name="/pose_stamped_topic"   />
    <node name = "$(arg node_name)" pkg = "orca_room" type = "pub_sub" output = "screen">
     <param name="target_model_name" type="string" value="$(arg target_model_name)" />
     <param name="target_model/twist_linear_x" type="double" value="0.5" />
     <param name="target_model/twist_angular_z" type="double" value="0.5" />
     <param name="neighbor_distance" type="double" value="2.0" />
     <param name="time_horizon" type="double" value="2.0" />
     <param name="radius_" type="double" value="1" />  
     <param name="goal_pose_x" type="double" value="$(arg goal_pose_x)" />
     <param name="goal_pose_y" type="double" value="$(arg goal_pose_y)" />
     <param name="goal_pose_z" type="double" value="$(arg goal_pose_z)" />
     <param name="time" type="double" value="0.1" />
     <param name="max_speed" type="double" value="3.0" />
     <remap from="/pose_stamped_topic" to="$(arg /pose_stamped_topic)" />
     <remap from="/path_topic" to="$(arg /path_topic)" />
    </node>

</launch>